### DISCUSSION TITLE
"Secure connection failed" error when authenticating to Google.

### DISCUSSION THREAD

**Discussion by JonathanRRogers** [kewisch/gdata-provider#266] [Support] [answered]
I'm using Provider for Google Calendar 91.0.1 on Thunderbird 91.2.0 on Ubuntu 20.04.3 LTS. A few days ago, it began failing to fully authenticate to my Google Calendar. Most steps succeed, but after I check boxes under "Select what Provider for Google Calendar can access" and click "Continue," I get a "Secure connection failed" page in the "Sign in to your account" window. There is an "Or you can add an exception..." link, but clicking it has no effect.

**Comment by Elleby77** [comment]
I have the same issue except on Windows 10.   So its not OS specific.

**Reply by Elleby77** [reply]
I just tried again (my fourth try today) and this time I got a completely different error message:

> 400. That’s an error.
> 
> The server cannot process the request because it is malformed. It should not be retried. That’s all we know.

I hope the developer can find a solution for this as this is a great tool that they made and I'd hate to see it become unusable.

**Reply by gmpreussner** [reply]
Same error here. Windows 10, Thunderbird 91.2.0 (32-bit), Provider for Google Calendar 91.0.1. I think it stopped working after the most recent Thunderbird upgrade.

**Reply by kewisch** [reply]
This might be fixed with the release candidate for Provider 91.0.2, please see #260 for details.

**Reply by gmpreussner** [reply]
I can confirm that the login dialogues and errors have disappeared with the RC. Updating folders seems a little sluggish now (takes up to 30 seconds where it was near instantaneous previously), but I'll keep an eye on it and report any new issues. Thanks!

**Reply by gmpreussner** [reply]
It looks like restarting Thunderbird resolved the sluggish syncs. I had to log into all my Google accounts one more time, but there were no errors.

**Reply by JonathanRRogers** [reply]
The release candidate for Provider 91.2.0 works for me.

**Comment by kewisch** [comment, marked as answer]
Noting for the answers that this should be fixed with 91.0.2 and above.
